packages/cloud_functions/cloud_functions/lib/src/https_callable.dart

```diff
@@ -48,6 +48,32 @@ class HttpsCallable {
     }
     return HttpsCallableResult<T>._(await delegate.call(updatedParameters));
   }
+
+  /// Streams data to the specified HTTPS endpoint.
+  ///
+  /// The data passed into the trigger can be any of the following types:
+  ///
+  /// `null`
+  /// `String`
+  /// `num`
+  /// [List], where the contained objects are also one of these types.
+  /// [Map], where the values are also one of these types.
+  ///
+  /// The request to the Cloud Functions backend made by this method
+  /// automatically includes a Firebase Instance ID token to identify the app
+  /// instance. If a user is logged in with Firebase Auth, an auth ID token for
+  /// the user is also automatically included.
+  Stream<StreamResponse> stream<T, R>([Object? input]) async* {
```

Comment:
In the API doc, there's HttpsCallableStreamOptions as an optional parameter, but I don't find the code in this PR. Anything changed?

Reply:
Yes, I had forgotten to update the documentation earlier. The original plan was to match the JS SDK’s `stream` signature, but we later decided to follow the native (Android/Java) convention instead. I’ve just updated the docs to reflect this change.